recipes/pyparam/meta.yaml

```
@@ -0,0 +1,46 @@
+{% set name = "pyparam" %}
+{% set version = "0.5.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2771512baa05c279d693a805233acf1894d4ad05b7f46b714ce74b1ac3ca96a8
+
+build:
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+  number: 0
+
+requirements:
+  host:
+    - python >=3.7
+    - pip
+    - poetry
+    - simpleconf
+  run:
+    - python >=3.7
+    - diot
+    - rich
+    - poetry
+    - simpleconf
```

```suggestion
    - python-simpleconf
```
Now your upstream [feedstock](https://github.com/conda-forge/python-simpleconf-feedstock) is created, you should point to that.

And, I don't think poetry is a runtime dependency?